storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -153,7 +153,12 @@ static bool reopen_log_files(lsn_t desired_lsn) {
     return false;
   }
 
-  ut_a(res == Log_files_find_result::FOUND_VALID_FILES);
+  if (res != Log_files_find_result::FOUND_VALID_FILES) {
```

Comment:
should we suggest our new redo log archiving option? (introduced in 8.0.34)?

Reply:
You mean changing the wording at `Consider using Redo Log Archiving` ?

Reply:
--redo-log-arch-dir=name

Reply:
Done